src/main.rs

```diff
@@ -57,6 +59,9 @@ struct Cli {
     /// List of file and/or directory patterns to exclude from checks.
     #[clap(long, multiple = true)]
     exclude: Vec<Pattern>,
+    /// Output formatting of linting messages
+    #[clap(long, value_name = "FORMAT", arg_enum, default_value_t=SerializationFormat::Text)]
```

Comment:
Is `value_name` here necessary? If so, what's it enforcing?

Reply:
It is not. Removed it!